iap/refresh/js/poll.js

```diff
@@ -0,0 +1,68 @@
+// Copyright Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+(function poll() {
+  setTimeout(function() {
+    var statusText = document.getElementById('status');
+    statusText.innerHTML = 'Polling';
+    var xhr = new XMLHttpRequest();
```

Comment:
Use `fetch`? https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API

Reply:
It has pretty wide support at this point https://caniuse.com/#feat=fetch

Reply:
Ok, sound's like the fetch/promise polyfill won't be needed then, I'll switch it over.

Reply:
Done, please check my promise usage, not sure if it is correct.